cpp/ql/test/library-tests/multiple_declarations/variable_types/extracted_once.h

```diff
@@ -0,0 +1,16 @@
+// This header file is extracted only once even though it's included by both
+// file1.c and file2.c. That's presumably because it's wrongly considered to
+// expand to the same trap in both contexts. In practice, this header gets
+// extracted together with the extraction of file1.c.
+
+// BUG: types of members depend on extraction order.
+// Only one copy of this struct is extracted, and the types of its members refer
+// to the typedefs in file1.c. Had file2.c been extracted first instead, the
+// types of its members would be different.
```

Comment:
That sounds right, and is similar to the way that the existing `variables/global` test would produce different output if `a.c` and `b.c` got extracted in a different order.

https://github.com/Semmle/ql/blob/88c74b2a4bb542f4c930bc5643eac3b11a358692/cpp/ql/test/library-tests/variables/global/variables.expected#L1-L2